bali/utils/timezone.py

```diff
@@ -75,5 +77,28 @@ def localtime(value: datetime = None, timezone: StrTzInfoType = None):
     return value.astimezone(timezone)
 
 
-def localdate(value: datetime = None, timezone: StrTzInfoType = None):
+def localdate(value: datetime = None, timezone: StrTzInfoType = None) -> date:
     return localtime(value, timezone).date()
+
+
+def start_of(
```

Comment:
What is `start_of ` used for?

Reply:
`start_of` is clearer and has more functions  then `get_beginning_datetime`

Reply:
and we can define `end_of` base on `start_of` in future